include/ignition/gui/qml/IgnSnackBar.qml

```diff
@@ -20,16 +20,24 @@ import QtQuick.Controls 2.2
 import QtQuick.Layouts 1.3
 import QtQuick.Window 2.2
 import QtQuick.Dialogs 1.0
+import QtQuick.Controls.Material 2.2
 
 Popup {
   id: snackbar
-  modal: duration == 0
-  focus: duration == 0
   x: (window.width - width) / 2
   y: window.height - window.height / 6
   width: window.width - window.width / 6
   contentHeight: notificationColumn.height
 
+  closePolicy: Popup.CloseOnEscape
```

Comment:
> ESC will close the popup, otherwise with the mouse click some of the popup will close inmediately if you are clicking several times to configure something.

I understand that this is meant to protect the user from closing the popup by mistake, but clicking outside it to close it is so common that I think users may be confused when that doesn't work (I was when I first tried this PR).

For popups that we really don't want the user to miss, how about supporting an optional "Dismiss" button, so the snackbar can only be dismissed by clicking that?